api/public/cerbos/svc/v1/svc.proto

```diff
@@ -73,6 +73,17 @@ service CerbosService {
             description: "Get information about the server e.g. server version"
         };
     }
+
+    rpc ResourcesQueryPlan(cerbos.request.v1.ResourcesQueryPlanRequest) returns (cerbos.response.v1.ResourcesQueryPlanResponse) {
+        option (google.api.http) = {
+            post: "/api/plan/resources",
```

Comment:
As discussed, let's put this under `/api/x/plan/resources` for now to indicate that it's experimental and there are no backward compatibility guarantees.